src/content.js

```diff
@@ -118,12 +118,19 @@ function moveMarketplaceLinkToProfileDropdown() {
 }
 
 async function addTrendingMenuItem() {
+	const selectedClass = pageDetect.isTrending() ? 'selected' : '';
 	const issuesLink = await safeElementReady('.HeaderNavlink[href="/issues"], .header-nav-link[href="/issues"]');
 	issuesLink.parentNode.after(
 		<li class="header-nav-item">
-			<a href="/trending" class="js-selected-navigation-item HeaderNavlink header-nav-link px-2" data-hotkey="g t">Trending</a>
+			<a href="/trending" class={`js-selected-navigation-item HeaderNavlink header-nav-link px-2 ${selectedClass}`} data-hotkey="g t">Trending</a>
 		</li>
 	);
+
+	// Explore link highlights /trending urls by default, remove that behavior
+	if (pageDetect.isTrending()) {
+		const exploreLink = await safeElementReady('a[href="/explore"]');
```

Comment:
Can you make this selector more specific?

Reply:
Something like this? `safeElementReady('.HeaderMenu a[href="/explore"], .header-menu a[href="/explore"])`. Guessing on the `header-menu` for GHE, but it'll probably be different there.

Reply:
Goes GHE also have Trending? :|

Also yeah let’s check the necessary classes for GHE first so you don’t have to send another PR next week 😃

Reply:
Yes, it has both options actually. GHE is easier with `'a.header-nav-link[href="/explore"]'`. I'll update the PR.

Reply:
`.catch(() => null)`